test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -50,6 +51,7 @@ public class DataAnnotationsMetadataProviderTest
                     { new ScaffoldColumnAttribute(scaffold: false), d => d.ShowForEdit, false },
 
                     { new UIHintAttribute("hintHint"), d => d.TemplateHint, "hintHint" },
+                    { new DisplayNameAttribute("DisplayNameValue"), d => d.DisplayName(), "DisplayNameValue"}
```

Comment:
nit: Think the attributes are in alphabetic order. If so, please move this up to where future Ryans will expect it.